### ISSUE TITLE
Adding a security measure(s)

### ISSUE THREAD

**Issue by Former-Orien** [jeroen1602/lighthouse_pm#134]
Lately, I've been noticing one of my basestations on after waking up from sleep or a nap. I get the feeling that one of my family members are currently using this app or a different one to tamper or turn them on when it's not theirs.

I believe a pin number should be implemented or once a basestation is registered to *only* the user who bought it can only control it. If these two options don't work then there should be an option to any or all basestations be visible ("hidden") to said registrant by Serial number.

**Comment by jeroen1602** [comment]
This app is made by a third party and not by Valve.

I cannot control the firmware that these base stations are running and thus can't implement some sort of pin code before being able to change the state. The Vive V1 base stations do require a pin before they work so that may be an option.

I could make some sort of database to "claim" base stations and then only allow the user that has claimed a base station to communicate with it. But I can't force other apps to also use this database before changing any state. It would also mean that I would need to maintain this database. For example what should happen if someone claims a base station and then sells it. Or someone claims some else's base station before they could.

In short I do agree that there is no security in the communication and that this opens it up for people to troll each other, but since I can't control the firmware of the base station I can't fix it on my end.

**Comment by Former-Orien** [comment]
Could we consider a serial number as a form of input for registering the basestation? Because if by "claiming" a basestation, the owner would also need a serial number to claim it. Thereby eliminating the potential for some stranger to claim/control someone property all because of an unsecured broadcast-wide bluetooth connection.
